keylime/cloud_verifier_common.py

```diff
@@ -57,7 +57,7 @@ def init_mtls(section='cloud_verifier', generatedir='cv_ca'):
 
         if generatedir[0] != '/':
             generatedir = os.path.abspath(
-                '%s/%s' % (config.WORK_DIR, generatedir))
+                f'{config.WORK_DIR}/{generatedir}')
```

Comment:
We should probably use `os.path.join()` instead of keep using format strings for file paths.

Reply:
good point